test/passes/extract-function_pass-arg=extract:foo.txt

```diff
@@ -0,0 +1,13 @@
+(module
```

Comment:
Are you embedding command line arguments in the file name?  Perhaps we could instead put these in command in the intput file, or in a separate ".run" file?

Driveby comment, no need to fix in this CL.

Reply:
We've been doing that for a while now yeah:
https://github.com/WebAssembly/binaryen/blob/65446aeade5b5a1acab6ae2a605ae1702251fda2/scripts/test/asm2wasm.py#L51-L54

Though looking for where we do: not in any one spot, oh dear